.ruby-version

```diff
@@ -1 +1 @@
-3.0
+3.0.3
```

Comment:
in the past, we'd left out the patch version from here, and had instructions in the README for folks to alias to the patch versions they have installed:

https://github.com/18F/identity-idp/blob/58d7154dfecb970cd823c46926eea2d95c0ca1d6/README.md#setting-up-and-running-the-app-without-docker

WDYT about bringing that approach and documenting it again?

Reply:
I think we do have to leave out the patch version, since it also provides flexibility in the deployed environments with how it's set up right now.

Reply:
The gemfile specifies `ruby '~> 3.0.3'` so I feel like the 3.0 is misleading since 3.0.0 will blow up. Also, when I enter the project it will switch my ruby version to 3.0.0, which will blow up. If I uninstall 3.0.0 I get the error:
```
Required ruby-3.0.0 is not installed.
To install do: 'rvm install "ruby-3.0.0"'
```
Is my environment handling rvm in a weird way or is `3.0` actually defining a patch version anyway? And one that's incompatible with our gemfile?

Reply:
Maybe it's an rvm issue? I use `rbenv` and an alias and things seem to work okay?

Reply:
I've also seen Gemfiles use `ruby RUBY_VERSION` which, when there's something like `rvm` or `rbenv` installed does the same thing as the `File.read` option

Reply:
@peggles2, This looks great! very dry and less personal config and aliasing to worry about. What about adding `~>` to the code like: `ruby "~> #{File.read(".ruby-version").strip}"` in the gemfile, so we can keep the semantic versioning to allow patches above the one listed. Or is that not necessary?

Reply:
I tested with `ruby RUBY_VERSION` and that didn't prevent me from using an older version or ruby.